src/lib/components/Tree/AddFrames.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+	import Drawer from './Drawer.svelte'
+	import { useFrames } from '$lib/hooks/useFrames.svelte'
+	import { usePartConfig } from '$lib/hooks/usePartConfig.svelte'
+
+	const frames = useFrames()
+	const partConfig = usePartConfig()
+</script>
+
+<Drawer name="Add Frames">
```

Comment:
```suggestion
<Drawer name="Add frames">
```
We prefer sentence casing for titles ✨